Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs

```diff
@@ -55,29 +63,27 @@ public void InitializeWithGenerateDelegatesOptionThrows()
         }
 #endif
 
-        [Fact]
+        [Fact, Obsolete]
```

Comment:
Ah. Nice! I was not aware of this attribute.

Reply:
This is a regular .NET `Obsolete` attribute. However, the trick is that you can call obsoleted code from the other obsoleted code without warnings, so it looks liner 😄